internal/core/input.rs

```diff
@@ -27,9 +27,9 @@ use core::pin::Pin;
 #[allow(missing_docs)]
 pub enum MouseEvent {
     /// The mouse or finger was pressed
-    Pressed { position: LogicalPoint, button: PointerEventButton },
+    Pressed { position: LogicalPoint, button: PointerEventButton, click_count: u8 },
```

Comment:
On second thought, why does the Pressed have the new field? Isn't it only relevant for the release?

Reply:
@tronical on text input selection is done on `pressed`.

Reply:
Oh, so the first time `click-count` is 1 is on `Release`, and the time it goes to `2` is the next press if it happens in time?

Reply:
if you press-click-press-click-press-click-press-click  4 times in a row, you'd have:

 - press 0
 - release 0
 - press 1
 - release 1
 - press 2
 - release 2
 - press 3
 - \<wait>
 - release 3   - this is contreversial should it be zero, i think it make sense to keep 3?
 - press 0
 - release 0
 - ....